app/Imports/XlsformTemplateImport.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace App\Imports;
+
+use App\Models\Language;
+use App\Models\SurveyRow;
+use App\Models\LanguageString;
+use App\Models\LanguageStringType;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Log;
+use App\Models\XlsformTemplateLanguage;
+use Maatwebsite\Excel\Concerns\ToCollection;
+use Maatwebsite\Excel\Concerns\WithHeadingRow;
+
+class XlsformTemplateImport implements ToCollection, WithHeadingRow
```

Comment:
We're going to need to split up the rows from the surveys worksheet and choices worksheet as they'll need to be handled differently when we come to rebuild the team's forms. So eventually we'll need this to be setup as a multiple worksheets import - but we can use basically the same approach for identifying translatable parts for both the survey and choice worksheets.